Crust/Utilities/ArrayAdaptor.swift

```diff
@@ -22,11 +22,11 @@ open class ArrayMapping<SubType: Equatable, SubAdaptor: Adaptor, SubMapping: Arr
     
     public var primaryKeys: [String : Keypath]? { return nil }
     open var keyPath: Keypath { return "" }
-    open var options: MappingOptions { return MappingOptions.None }
+    open var options: CollectionUpdatePolicy<SubMapping.SequenceKind> { return (.append, true) }
```

Comment:
I stated this in another comment - I'm going to delete all this Array Adaptor shit. Will do in a subsequent PR.